platform/src/main/java/org/stellar/anchor/platform/config/FireblocksConfig.java

```diff
@@ -191,6 +191,7 @@ public void validateRetryDelay(Errors errors) {
    */
   public PublicKey getFireblocksPublicKey() throws InvalidConfigException {
     try {
+      System.out.println("publicKey=" + publicKey);
```

Comment:
Nit: remove println

Reply:
I was working on removing all the printlin statements too. :-)